queries.sql

```diff
@@ -8,3 +9,45 @@ SELECT NAME, ESCAPE_ATTEMPTS FROM animals WHERE WEIGHT_KG>10.5;
 SELECT * FROM animals WHERE NEUTERED=true;
 SELECT * FROM animals WHERE NAME!='Gabumon';
 SELECT * FROM animals WHERE WEIGHT_KG BETWEEN 10.4 AND 17.3;
+
+/* Task Two */
+
+BEGIN;
+
+UPDATE animals SET SPECIES='unspecified';
+
+ROLLBACK;
```

Comment:
- [ ] Please ensure you add a query to verify the change before and after rollback as shown below 👇🏾 

```sql
   BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone
```